model/pod/db.go

```diff
@@ -66,15 +66,15 @@ func UpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
-// Find gets all Pods for the given query.
-func Find(query db.Q) ([]Pod, error) {
+// Find gets all Pods for the given BSON document.
```

Comment:
I would leave the comment as it was originally. This is actually not a BSON document type - it's a map. If you want a BSON document, you would use `bson.D`. The main difference between `bson.M` and `bson.D` is that the ordering of the keys in a `bson.D` matters and keys can repeat, while it doesn't matter for `bson.M` and keys are unique.